src/studio/README.md

```diff
@@ -113,12 +114,18 @@ On MacOS you need two extra steps:
    export ALTINN_KEYS_DIRECTORY=/Users/<yourname>/studio/keys
    ```
 
-Build and run the code.
+Build and prepare for running the application
 
 ```bash
 dotnet build
 yarn run gulp # first time only
-yarn run gulp-develop
+```
+
+There are multiple ways to start the applications
```

Comment:
Sorry, approved instead of adding my comment. No biggie, but if I'm not mistaken these two commands are the only way to get updated files in the wwwroot/designer/js/react folder which in turn will be served when running designer from Visual Studio. If there are changes in both applications I think you need to run both in order to get app-development.js and dashboard.js updated.

Reply:
Both of those commands also start the backend. I honestly just assumed that made them mutually exclusive. There are other commands that start the front end without backend. All commands hooks up a watcher for some part of the code.

Reply:
Yes, both start backend but they also make sure that updated files are served from the backend. I made the mistake of thinking that running the first was enough only to find that the dashboard changes I was waiting for wasn't included.

Reply:
To get updated files served from backend, the gulp task "copy-files" needs to be executed (after the frontend has been built). The easiest way to do this is to execute `yarn run gulp`, which will run the default gulp task. The default gulp task is set up to build both app-frontend and dashboard, and finally copy the files to backend so they can be served.

In the readme it say `yarn run gulp # first time only`, which is not really correct - it should be run whenever there are frontend changes, and you want to see those changes when serving those via the backend... A bit confusing stuff 🙃